src/clj/game/cards/ice.clj

```diff
@@ -1060,6 +1060,12 @@
    "Rainbow"
    {:subroutines [end-the-run]}
 
+   "Ravana 1.0"
+   {:subroutines [{:label "Resolve a subroutine on another piece of rezzed bioroid ice."
+                   :choices {:req #(and (rezzed? %) (ice? %) (has-subtype? % "Bioroid"))}
+                   :msg (msg "Resolve subroutine on " (:title target))}]
```

Comment:
Make "Resolve" all lowercase and put "a" in between it and "subroutine". Currently your message will come out in the log as:  "Playername uses Ravana 1.0 to Resolve subroutine on ICE-name-here"